Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.8.1-slim
+
+WORKDIR /app
+ARG requirements=requirements/production.txt
+
+COPY ./$requirements /app/$requirements
+RUN pip install --no-cache-dir -r $requirements
+
+COPY . /app
+
+RUN pip install --no-cache-dir -e .
+
+ENTRYPOINT ["nil", "run"]
```

Let's use the `CMD` instead of the `ENTRYPOINT`. This way it will be easier to add a custom `docker-entrypoint.sh` with a pre-run logic while keeping all commands the same and make sure it is run for every script inside the container.

Also, `docker-compose run node *some command*` will invoke `nil run *some command*` in the `node` container and will fail.

---

In next PR - we will entrypoint. Too much for this.

---

You are right, the addition of the entrypoint script is out of the scope of this PR. I'm not asking to add one, only to change the image's entrypoint into a command instead:
```suggestion
CMD ["nil", "run"]
```

But this is not critical and could be done alongside `docker-entrypoint.sh` addition in another PR.